src/main/java/model/Money.java

```diff
@@ -0,0 +1,41 @@
+package model;
+
+import java.math.BigDecimal;
+
+import static model.Lotto.LOTTO_PRICE;
+
+public class Money {
+    private static final String MONEY_MUST_BE_POSITIVE = "구입금액은 0 이상의 숫자여야 합니다.";
+    private static final int IS_NEGATIVE = -1;
+
+    private final BigDecimal amount;
+
+    public Money(BigDecimal amount) {
+        validateAmount(amount);
+        this.amount = amount;
+    }
+
+    private void validateAmount(BigDecimal amount) {
+        boolean isNegativeAmount = amount.compareTo(BigDecimal.ZERO) == IS_NEGATIVE;
+        if (isNegativeAmount) {
```

Comment:
약간 과할 수도 있는 리뷰일 수 있으나, 저라면 이렇게 변경했을 거 같아요! 
```suggestion
        int status = amount.compareTo(BigDecimal.ZERO);
        if (status  == IS_NEGATIVE) {
```

Reply:
좀 더 가독성이 좋아진 것 같습니다. 감사합니다.